tools/copy_bom/src/util.rs

```diff
@@ -598,3 +595,17 @@ pub fn warn_on_nonempty_image_dir(image_dir: &str) {
         }
     }
 }
+
+/// Check rsync is installed by running command `rsync --version`. 
+/// The exit code should be zere if rsync is installed.
+/// If rsync is not installed, copy_bom will abort.
+pub fn check_rsync()  {
+    let mut command = Command::new("rsync");
+    command.arg("--version");
+    let status = command.status().expect("execute `rsync --version` failed");
```

Comment:
Is the rsync info printed out when the user running the copy_bom tool? Should we use "rsync --version > /dev/null"?

Reply:
Yes, it will print to console. Fixed now.

Reply:
If rsync is not installed, copy_bom will panic. Should it output the message "rsync is not installed" and then exit the process with the `RSYNC_NOT_FOUND_ERROR` exitcode instead of having a panic?

Reply:
Fixed now.